test/graphql/types/Organization/creator.test.ts

```
@@ -0,0 +1,389 @@
+    );
+    expect(result).toEqual(mockCreator);
+  });
+
+  it("should verify columns are correctly specified in the user query", async () => {
+    const mockUser = {
+      role: "administrator",
+      organizationMembershipsWhereMember: [],
+    };
+
+    let columnsVerified = false;
+    ctx.drizzleClient.query.usersTable.findFirst.mockImplementationOnce(async (params: { columns: any; }) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)